docs/workflow/workflow.md

```diff
@@ -0,0 +1,80 @@
+# Workflow Details
+
+This section describes the SCATTR workflow (i.e. steps taken to produce 
+intermediate and final files). SCATTR is a 
+[Snakemake](https://snakemake.readthedocs.io/en/stable/) workflow, and thus a 
+directed acyclic graph (DAG) that is automatically configured based on a set of 
+rules.
+
+## Overall workflow
+
+Below is an image exhibiting the workflow DAG. Each rounded rectangle in 
+the DAG represents a rule (i.e. some code or script that produces an output), 
+with arrows representing the connections (i.e. inputs / outputs) to these rules.
+
+<img src="dag.png" width="800px">
+_(Click on the image to enlarge)_
```

Comment:
Would you be able to grab a screenshot of what you mean here?

Reply:
![Screenshot 2023-04-20 at 13-25-23 Workflow Details — SCATTR documentation](https://user-images.githubusercontent.com/6700252/233442301-460675af-fa84-4577-a37c-d71e53a18fd2.png)

Reply:
Hmm I am at a loss for this one - did I put the path in wrong? I tried to change it to a relative path, but that doesn't seem like it fixed it either.

Reply:
I'm not sure either. I'd just get rid of the underscores to keep it simple

Reply:
It doens't like relative paths (maybe something that needs to be enabled). In any case, I tried it with the path of where the image is and it shows up okay:

![image](https://user-images.githubusercontent.com/21226986/233607398-c8b12a8c-1695-43dc-a86e-69d59a42707b.png)

I've removed the stuff with the underscores (enlarging) since this is already pretty easy to read. I've also updated the path in the latest push to where the image is expected to be (on the master branch) so it may not render anymore on the latest build. Can check out this [build](https://readthedocs.org/projects/scattr/builds/20194498/) for the rendering of the image.